src/plugins/recordTypes/media/fields.js

```diff
@@ -0,0 +1,151 @@
+          [config]: {
+            required: false,
+            view: {
+              type: TextInput,
+            },
+          },
+        },
+        title: {
+          [config]: {
+            defaultValue: 'media record',
+            view: {
+              type: OptionPickerInput,
+              props: {
+                source: 'mediaTitles',
+              },
+            },
+          },
+        },
+        languageList: {
+          language: {
+            [config]: {
+              defaultValue: 'urn:cspace:pahma.cspace.berkeley.edu:vocabularies:name(languages):item:name(eng)\'English\'',
+            },
+          },
+        },
+        typeList: {
+          type: {
+            [config]: {
+              view: {
+                props: {
+                  source: 'pahmaMediaTypes',
+                },
+              },
+            },
+          },
+        },
+        coverage: {
+          [config]: {
+            view: {
+              type: AutocompleteInput,
+              props: {
+                source: 'place/local',
+              },
+            },
+          },
+        },
+        subjectList: {
+          subject: {
+            [config]: {
+              view: {
+                type: AutocompleteInput,
+                props: {
+                  source: 'place/local,concept/ethculture,concept/archculture,person/local,organization/local,taxon/local,concept/activity,concept/material',
+                },
+              },
+            },
+          },
+        },
+      },
+      'ns2:media_pahma': {
+        [config]: {
+          service: {
+            ns: 'http://collectionspace.org/services/media/local/pahma',
+          },
+        },
+        approvedForWeb: {
+          [config]: {
+            defaultValue: 'true',
+            dataType: DATA_TYPE_BOOL,
+            messages: defineMessages({
+              name: {
+                id: 'field.media_pahma.approvedForWeb.name',
+                defaultMessage: 'Approved for public?',
+              },
+            }),
+            view: {
+              type: CheckboxInput,
+            },
+          },
+        },
+        primaryDisplay: {
+          [config]: {
+            dataType: DATA_TYPE_BOOL,
+            messages: defineMessages({
+              name: {
+                id: 'field.media_pahma.primaryDisplay.name',
+                defaultMessage: 'Primary display?',
+              },
+            }),
+            view: {
+              type: CheckboxInput,
+            },
+          },
+        },
+        documentationImage: {
+          [config]: {
+            dataType: DATA_TYPE_BOOL,
+            messages: defineMessages({
+              name: {
+                id: 'field.media_pahma.documentationImage.name',
+                defaultMessage: 'Documentation image?',
+              },
+            }),
+            view: {
+              type: CheckboxInput,
+            },
+          },
+        },
+        documentationImageSearch: {
```

Comment:
Don't need this field. In 4.1 we needed to configure a separate field to search on booleans, but this isn't needed in 5.0.